### ISSUE TITLE
Hex sticker ideas

### ISSUE THREAD

**Issue by jvcasillas** [opnlnx/opnlnx.github.io#2]
![opnlnx_1](https://user-images.githubusercontent.com/1747068/68521679-72df0580-0271-11ea-990b-1a72ba437146.png)

![opnlnx_2](https://user-images.githubusercontent.com/1747068/68521680-72df0580-0271-11ea-868a-6dff67f13a2f.png)

**Comment by jvcasillas** [comment]
Some thoughts: I like the simplicity of the hex stickers (I'm a minimalist), but I feel like there may be too much of a disconnect between OpnLnx and Open Language Sciences. What about just OpenLang? It seems more transparent that one comes from the other. OpenLangSci. OpnLngSci. Hmm.

**Comment by troettge** [comment]
Yeah LangSci we cannot use because of the LangSci book series. OpnLnx corresponds directly with Open Linguistics which is straightforward. Its just having been in the states, linguistics here is considered a  particular kind of language sciences that is "theoretical" in a particular sense. Language sciences seems a little more inclusive. But I totally see the disconnect. hm...

**Comment by jvcasillas** [comment]
I haven't thought of any other options, but I will post something if I do.
